pandas/core/window.py

```diff
@@ -899,11 +899,45 @@ def skew(self, **kwargs):
         return self._apply('roll_skew', 'skew',
                            check_minp=_require_min_periods(3), **kwargs)
 
-    _shared_docs['kurt'] = """Unbiased %(name)s kurtosis"""
+    _shared_docs['kurt'] = dedent("""Calculate unbiased %(name)s kurtosis.
 
-    def kurt(self, **kwargs):
+    This function uses Fisher's definition of kurtosis (kurtosis of normal
+    == 0.0) without bias.
+
+    Returns
+    -------
+    same type as input
+
+    See Also
+    --------
+    scipy.stats.kurtosis
+    pandas.DataFrame.kurtosis
+    pandas.Series.kurtosis
+
+    Notes
+    -----
+    A minimum of 4 periods is required for the rolling calculation
```

Comment:
Nice, did you find that the documentation about the "Notes" section made sense for this case?

Reply:
I added this from familiarity with the NumPy standard, not necessarily the docs that you built (although they were *very* helpful on many other aspects - kudos)